akka-http/src/main/resources/reference.conf

```diff
@@ -39,5 +39,5 @@ akka.http.routing {
 
   # Fully qualified config path which holds the dispatcher configuration
   # to be used by FlowMaterialiser when creating Actors for IO operations.
-  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher}
+  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} // TODO point to akka.stream.file-io-dispatcher instead?
```

Comment:
@sirthias @jrudolph @patriknw any feelings on this suggestion? I'm asking because `akka.io.tcp.file-io-dispatcher` is by default the default dispatcher of the actor system https://github.com/akka/akka/blob/release-2.3-dev/akka-actor/src/main/resources/reference.conf#L574 and users are expected to configure one by themselfes for IO. For the file-source / file-sink we now provide a preconfigured simple threadpool dispatcher: `akka.stream.file-io-dispatcher`

Should we keep this setting as is here?